src/features/landing/components/AboutVilnaSection/AboutVilnaBackgroundWithSecondarySvg/AboutVilnaBackgroundWithSecondaryPng.tsx

```diff
@@ -0,0 +1,30 @@
+import { Box } from '@mui/material';
+import React from 'react';
```

Comment:
this component look like AboutVilnaBackgroundWithMainSvg.tsx, read https://ru.wikipedia.org/wiki/Don%E2%80%99t_repeat_yourself and after refactor all of this